npm/builder.js

```diff
@@ -481,8 +479,8 @@ Builder.prototype.compile = function() {
 
   log.title('copy resources');
   log.debug('copy asciidoctor.css');
-  // FIXME: Find a more robust implementation to get the Asciidoctor gem install path
-  var asciidoctorGemPath = child_process.execSync("ruby -e \"print (Gem::Specification.find_by_name 'asciidoctor').full_gem_path\"");
+  // REMIND: Asciidoctor is now a Git submodule
+  var asciidoctorGemPath = 'asciidoctor';
```

Comment:
I think we should build against a release instead of master. Can we just download the zip from GitHub?